halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/ui/v1/Ui.java

```diff
@@ -56,6 +52,10 @@ public void failure(String message) {
     reset(color.BOLD + color.RED + "! " + color.RESET + message);
   }
 
+  public void remediation(String message) {
```

Comment:
You read my mind - while writing this I realized I should change up my error/warning reporting to also supply possible remediations wherever possible.